packages/prerender/src/runPrerender.tsx

```diff
@@ -19,41 +17,36 @@ interface PrerenderParams {
 
 const rwWebPaths = getPaths().web
 
-// Prerender specific configuration
-// extends projects web/babelConfig
-babelRequireHook({
-  extends: path.join(rwWebPaths.base, '.babelrc.js'),
-  extensions: ['.js', '.ts', '.tsx', '.jsx'],
-  overrides: [
-    {
-      plugins: [
-        ['ignore-html-and-css-imports'], // webpack/postcss handles CSS imports
-        [
-          'babel-plugin-module-resolver',
-          {
-            alias: {
-              src: rwWebPaths.src,
-            },
-            root: [getPaths().web.base],
-            // needed for respecting users' custom aliases in web/.babelrc
-            // See https://github.com/tleunen/babel-plugin-module-resolver/blob/master/DOCS.md#cwd
-            cwd: 'babelrc',
-            loglevel: 'silent', // to silence the unnecessary warnings
-          },
-          'prerender-module-resolver', // add this name, so it doesn't overwrite custom module resolvers in users' web/.babelrc
-        ],
-        [mediaImportsPlugin],
-      ],
-    },
-  ],
-  only: [getPaths().base],
-  ignore: ['node_modules'],
-  cache: false,
-})
-
 export const runPrerender = async ({
   routerPath,
 }: PrerenderParams): Promise<string | void> => {
+  // Prerender specific configuration
+  // extends projects web/babelConfig
+  registerWebSideBabelHook({
+    overrides: [
+      {
+        plugins: [
+          ['ignore-html-and-css-imports'], // webpack/postcss handles CSS imports
+          [
+            'babel-plugin-module-resolver',
+            {
+              alias: {
+                src: rwWebPaths.src,
+              },
+              root: [getPaths().web.base],
+              // needed for respecting users' custom aliases in web/.babelrc
+              // See https://github.com/tleunen/babel-plugin-module-resolver/blob/master/DOCS.md#cwd
+              cwd: 'babelrc',
+              loglevel: 'silent', // to silence the unnecessary warnings
+            },
```

Comment:
Does the rw alias plugin I wrote not work in this case?

Reply:
I suggest we don't use that here for now - because there's a chance people may have custom aliases setup (like Burnsy). 

My suggestions: keep as is, and in a later PR - we add support for specifying your own aliases, and ask users to migrate to this new system.

Reply:
Maybe we can be clever, and just read their tsconfig/jsconfig instead of them having to specify it somewhere else.

Reply:
Using ours would not invalidate them using theirs.